[2024-07-14T23:50:50.625+0100] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-14T23:50:50.685+0100] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: morning.get_sentiments manual__2024-07-14T22:49:29.715856+00:00 [queued]>
[2024-07-14T23:50:50.693+0100] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: morning.get_sentiments manual__2024-07-14T22:49:29.715856+00:00 [queued]>
[2024-07-14T23:50:50.694+0100] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-14T23:50:50.708+0100] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): get_sentiments> on 2024-07-14 22:49:29.715856+00:00
[2024-07-14T23:50:50.722+0100] {logging_mixin.py:188} WARNING - /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=15884) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-14T23:50:50.723+0100] {standard_task_runner.py:63} INFO - Started process 15905 to run task
[2024-07-14T23:50:50.724+0100] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'morning', 'get_sentiments', 'manual__2024-07-14T22:49:29.715856+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/var/folders/ml/zd6cs33j1gs1prd5lpns9pnc0000gn/T/tmpu5yqexn9']
[2024-07-14T23:50:50.727+0100] {standard_task_runner.py:91} INFO - Job 5: Subtask get_sentiments
[2024-07-14T23:50:50.787+0100] {task_command.py:426} INFO - Running <TaskInstance: morning.get_sentiments manual__2024-07-14T22:49:29.715856+00:00 [running]> on host MAKs-MacBook-Pro.local
[2024-07-14T23:50:50.994+0100] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='morning' AIRFLOW_CTX_TASK_ID='get_sentiments' AIRFLOW_CTX_EXECUTION_DATE='2024-07-14T22:49:29.715856+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-14T22:49:29.715856+00:00'
[2024-07-14T23:50:50.997+0100] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-14T23:50:54.960+0100] {python.py:237} INFO - Done. Returned value was:            ds  negative  positive  ...   sadness  surprise    others
0  2024-07-14  0.132998  0.102308  ...  0.000831  0.003496  0.977262

[1 rows x 11 columns]
[2024-07-14T23:50:54.979+0100] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-14T23:50:55.003+0100] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=morning, task_id=get_sentiments, run_id=manual__2024-07-14T22:49:29.715856+00:00, execution_date=20240714T224929, start_date=20240714T225050, end_date=20240714T225055
[2024-07-14T23:50:55.058+0100] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-14T23:50:55.081+0100] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-14T23:50:55.083+0100] {local_task_job_runner.py:222} INFO - ::endgroup::
